courses/free-courses-te.md

```diff
@@ -1,39 +1,204 @@
-### Index
-
+# Index
+
+* [Android](#android)
+* [Angular](#angular)
+* [Artificial Intelligence](#artificial-intelligence)
+* [Automata Theory](#automata-theory)
+* [Bash and Shell](#bash-and-shell)
+* [Bootstrap](#bootstrap)
 * [C](#c)
-* [C++](#cpp)
+* [C++](#c-1)
+* [Cloud Computing](#cloud-computing)
+* [Compiler Design](#compiler-design)
+* [CSS](#css) 
+* [Database Management Systems](#database-management-systems)
+* [Data Science](#data-science)
+* [Data Structures](#data-structures)
+* [Ethical Hacking](#ethical-hacking)
+* [ExpressJS](#expressjs)
+* [HTML](#html)
 * [Java](#java)
 * [Javascript](#javascript)
+* [Laravel](#laravel)
+* [Machine Learning](#machine-learning)
+* [MongoDB](#mongodb)
+* [NextJS](#nextjs)
+* [NodeJS](#nodejs)
+* [PHP](#php)
 * [Python](#python)
 * [R](#r)
+* [ReactJS](#reactjs)
+* [SQL](#sql)
+* [VueJs](#vuejs)
+
+
+### Android
+
+* [Android Development course in 2 Hours in Telugu](https://www.youtube.com/watch?v=7ZLKpN8vXLo) - Python Life
+* [Android App Development Tutorials For Beginners in Telugu Android Studio](https://www.youtube.com/playlist?list=PLeCpoxUq3EhmgHp7KI-Ih3X2PcNYRtC5y) - Sai Gopi
+
+
+### Angular
+
+* [Angular In Telugu | ANGULAR 10 IN TELUGU |angular in telugu|ANGULAR INTRODUCTION FOR BEGINNERS(2021)](https://www.youtube.com/watch?v=9MxS8oNlnMM) - TeluguTechSteps
+* [Angular In Telugu](https://www.youtube.com/playlist?list=PLO7Oa5iXf4QhtPXkaNX05qhGQSKFsvAF7) - TeluguTechSteps
+
+
+### Artificial Intelligence
+
+* [Artificial Intelligence 12 Hours Course In Telugu](https://www.youtube.com/watch?v=fwMMBaXIpqQ) - Python Life
+
+
+### Automata Theory
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.